solr/solrj/src/test/org/apache/solr/client/ref_guide_examples/ZkConfigFilesTest.java

```diff
@@ -38,12 +37,13 @@
  */
 public class ZkConfigFilesTest extends SolrCloudTestCase {
 
-  private static final int ZK_TIMEOUT_MILLIS = 10000;
+  private static ConfigSetService configSetService;
```

Comment:
A static field (that isn't a constant) like this needs to be null'ed in AfterClass for a test.  It may hold onto resources longer than we want; maybe even leading to problems.

Alternatively, let's add a `getConfigSetService()` method to the MiniSolrCloudService that makes this convenient?  Only if it might be used in a number of places.

Reply:
I added `getCoreContainer()` to MiniCloudServer